### PR TITLE
Add default metrics supported by OpenCensus system metrics package

### PR DESCRIPTION
**Pull request by mayurkale22**
This updates: #225

### CODE REVIEW

File: metrics/SystemMetrics.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+| Metric Name                               | Unit  | Type                       | Description                                                                                                | Labels                 |
+|-------------------------------------------|-------|----------------------------|-----------------------------------------------------------------------------------------------|-----------------|
+| system/cpu_seconds/total                  | s     | Int64 Cumulative           | Total kernel/system user CPU in seconds.                                                          | Hostname, Mode          |
+| system/processes/created                  | 1     | Int64 Cumulative           | Total number of times a process was created.                                                      | Hostname                |
+| system/processes/running                  | 1     | Int64 Gauge                | Total number of running processes.                                                                | Hostname                |
```
Comment (rghetia) [inline metrics/SystemMetrics.md]: add blocked process
system/processes/blocked, 1, Int64 Gauge, Number of processes blocked.

File: metrics/SystemMetrics.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+| system/disk/total                         | By    | Int64 Gauge                | The total available system disk space in bytes.                                                   | Hostname, Volume        |
+| system/disk/used                          | By    | Int64 Gauge                | The total used system disk space in bytes.                                                        | Hostname, Volume        |
```
Comment (rghetia) [inline metrics/SystemMetrics.md]: add memory related stats
system/memory/used
system/memory/total
system/memory/free
all of them are Int64 Gauges.

File: metrics/SystemMetrics.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+| process/heap/used                         | By    | Int64 Gauge                | The process's total used heap size in bytes.                                                      | Hostname                |
+| process/memory/rss                        | By    | Int64 Gauge                | Resident memory size in bytes.                                                                    | Hostname                |
+| process/file_descriptor/max               | 1     | Int64 Gauge                | The maximum limit of file descriptor count.                                                       | Hostname                |
+| process/file_descriptor/open              | 1     | Int64 Gauge                | The number of open file descriptor count.                                                         | Hostname                |
+| process/start_time                        | s     | Int64 Cumulative           | The start time of the process since unix epoch in seconds.                                        | Hostname                |
```
Comment (rghetia) [inline metrics/SystemMetrics.md]: This has no meaning. As it will be constant. May be a  wall-clock time would be better.

File: metrics/SystemMetrics.md (hunk 1 above)
Comment (mayurkale22) [reply metrics/SystemMetrics.md]: Done, thanks.

File: metrics/SystemMetrics.md (hunk 2 above)
Comment (mayurkale22) [reply metrics/SystemMetrics.md]: Sure, added.

File: metrics/SystemMetrics.md (hunk 3 above)
Comment (mayurkale22) [reply metrics/SystemMetrics.md]: Good catch, removed. Thx

File: metrics/SystemMetrics.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Supported platforms
+TODO: Add list of supported platforms.
+
+## Common metrics collection
+These are some default metrics supported by OpenCensus system metrics package/contrib. This is to ensure consistency across language implementations, as far as practical.
+
+| Metric Name                               | Unit  | Type                       | Description                                                        | Labels                  |
+|-------------------------------------------|-------|----------------------------|--------------------------------------------------------------------|-------------------------|
+| system/cpu_seconds/total                  | s     | Int64 Cumulative           | Total kernel/system user CPU in seconds.                           | Hostname, Mode          |
```
Comment (bogdandrutu) [inline metrics/SystemMetrics.md]: @rghetia can you add your idea about the idle/total/user label idea?

Comment (rghetia) [reply metrics/SystemMetrics.md]: It is already included here but I would rephrase it differently.

system/cpu_seconds  | s  | Int64 Cumulative |  System CPU in seconds | Hostname, Mode = { total, idle, system, nice, user, iowait, stolen}

Similarly other metrics can be redefined with common metric and possible Label and its values.

File: metrics/SystemMetrics.md [hunk 5]
```diff
@@ -7,27 +7,30 @@ TODO: Add list of supported platforms.
...
+| process/heap/used                         | By    | Int64 Gauge                | The process's total used heap size in bytes.                       | Hostname                |
+| process/cpu/usage                         | s     | Int64 Cumulative           | The total user and system CPU time spent in seconds.               | Hostname                |
 
 ### Labels
 The label keys associated with the above metrics.
 
 | Label Key          | Description                                                                                              |
 |--------------------|----------------------------------------------------------------------------------------------------------|
-| Mode               | The amount of time spent by the CPU in state (i.e. `user, nice, system, idle, iowait`).                  |
+| Mode               | The amount of time spent by the CPU in state (i.e. `user, nice, system, idle, iowait, stolen`).          |
```
Comment (rghetia) [inline metrics/SystemMetrics.md]: Since the label value is related to metric it might be better to put that in the label column rather than here.

File: metrics/SystemMetrics.md (hunk 4 above)
Comment (mayurkale22) [reply metrics/SystemMetrics.md]: Sure, done.

File: metrics/SystemMetrics.md (hunk 5 above)
Comment (mayurkale22) [reply metrics/SystemMetrics.md]: The Label is a combination of key and description, to avoid unnecessary duplication in metrics table I have added separate table for labels. Let me know if you still want me to change.
